tests/packages/extra/conftest.py

```diff
@@ -0,0 +1,12 @@
+import logging
+import pytest
+import urllib.request
+
+@pytest.fixture(scope="session")
+def extra_pkgs(host):
+    version = host.xcp_version
+    url = f"https://reports.xcp-ng.org/{version}/extra_installable.txt"
+
+    logging.info(f"getting extra packages from {url}")
```

Comment:
Most our logs begin with a capital letter in this project.

```suggestion
    logging.info(f"Getting extra packages from {url}")
```